code/game/objects/items/devices/powersink.dm

```diff
@@ -78,6 +78,10 @@
 				"[user] detaches \the [src] from the cable.", \
 				"<span class='notice'>You detach \the [src] from the cable.</span>",
 				"<span class='italics'>You hear some wires being disconnected from something.</span>")
+	// yogs start - multitool for power drained
+	if(istype(I, /obj/item/multitool))
+		to_chat(user, "The power dial reads [power_drained] kW.")

```

Comment:
Shouldn't this be joules

Reply:
Using a multitool on a wire uses kW so I just used that unit for this.

Reply:
a watt is one joule per second, this is like saying you have 2 liters per second of water in a bottle